android/app/proguard-rules.pro

```diff
@@ -9,8 +9,17 @@
 
 # Add any project specific keep options here:
 
-# react-native-config, prevents obfuscating of .env flags
--keep class org.pathcheck.covidsafepaths.BuildConfig { *; }
+-keepattributes LineNumberTable,SourceFile
+
+# This allows proguard to strip isLoggable() blocks containing only <=INFO log
+# code from release builds.
+-assumenosideeffects class android.util.Log {
+  public static *** d(...);
+  public static *** v(...);
+  public static *** isLoggable(...);
+}
```

Comment:
Logs are removed from staging/release builds.
I can add them only for staging if we find it useful for QA in the future